pom.xml

```diff
@@ -624,6 +624,8 @@
                 <exclude>website/**</exclude>
                 <exclude>.github/*</exclude>
                 <exclude>**/*.properties</exclude>
+                <!-- This directory onyl exists on travis -->
+                <exclude>viennacl-dev-release-1.7.1/**</exclude>
```

Comment:
this is the directory that the ViennaCL C++ headers are cloned into to be built against with  JavaCPP`, here, in travis
.